default.nix

```diff
@@ -1,8 +1,11 @@
-{ obelisk ? import ./.obelisk/impl (builtins.removeAttrs args ["pkgs"])
+{ config ? {}
+, obelisk ? import ./.obelisk/impl (builtins.removeAttrs args ["pkgs"])
 , pkgs ? obelisk.nixpkgs
 , ... } @ args:
 
 let
+  obelisk = import ./.obelisk/impl { inherit config; };
+  pkgs = obelisk.nixpkgs;
```

Comment:
This looks weird. We are defining both `obelisk` and `pkgs` twice?

Reply:
Bizarre. Maybe a merge went wrong?

Reply:
Very likely.